examples/hmac.rs

```diff
@@ -0,0 +1,64 @@
+#![no_std]
+
+use core::fmt::Write;
+use libtock::debug::print_as_hex;
+use libtock::hmac::{HmacDataBuffer, HmacDestBuffer, HmacKeyBuffer};
+use libtock::result::TockResult;
+use libtock::syscalls;
+
+#[libtock::main]
+async fn main() -> TockResult<()> {
+    let mut drivers = libtock::retrieve_drivers()?;
+
+    let mut console = drivers.console.create_console();
+
+    writeln!(console, "Starting HMAC example")?;
+
+    let hmac_driver = drivers.hmac.init_driver()?;
+
+    writeln!(console, "Loading in 0 key")?;
+    let mut key_buffer = HmacKeyBuffer::default();
+    let _key_buffer = hmac_driver.init_key_buffer(&mut key_buffer)?;
+    writeln!(console, "  done")?;
+
+    writeln!(console, "Creating data buffer")?;
+    let mut data_buffer = HmacDataBuffer::default();
+    let data = [
```

Comment:
You can use something like `let data: [u8;??] = b"A language"`.

Reply:
That's helpful :)

Reply:
I think you still need to push this change.

Reply:
Just pushed